frontend/src/app/AdminNavBar/AdminNavBar.tsx

```diff
@@ -143,6 +219,12 @@ export const AdminNavBar = ({ isMenuOpen }: AdminNavBarProps): JSX.Element => {
           {NAV_LINKS.map((link, index) => (
             <AdminNavBarLink key={index} {...link} />
           ))}
+          <WhatsNewNavBarTab
+            label={WHATS_NEW_LABEL}
+            onClick={onWhatsNewDrawerOpen}
+            MobileIcon={BxsRocket}
```

Comment:
should we inline this MobileIcon prop also? not sure why we are passing in this as a prop